source/sitemap.xml.erb

```diff
@@ -41,9 +41,9 @@
 
     <% data.locales.each do |locale| %>
       <% if locale[:id] == default_locale_id %>
-        <xhtml:link rel="alternate" href="<%= localize_url(category[:path], locale_id: locale[:id]) %>" hreflang="x-default" />
+        <xhtml:link rel="alternate" href="<%= localize_url(category[:path], locale_id: locale[:id]) %>" />
```

Comment:
Assuming this is what the story said 👍   I did zero investigation into the technical changes, so just assuming we're good here.